src/encoded/static/components/genetic_modification.js

```diff
@@ -693,7 +719,7 @@ GeneticModificationSummary.columns = {
     method: { title: 'Method' },
     site: {
         title: 'Site',
-        display: item => <ModificationSiteItems geneticModification={item} itemClass={'gm-table-modification-site'} />,
+        display: item => (hasModificationSiteProps(item) ? <ModificationSiteItems geneticModification={item} itemClass={'gm-table-modification-site'} /> : null),
```

Comment:
Not big deal
Am I wrong, or does line have code you repeated in line 149 hasModificationSiteProps(...)? It may be clearer to put hasModificationSiteProps() inside ModificationSiteItems. The component ModificationSiteItems is called and the caller let ModificationSiteItems worry about whether or not it can display